python/az/aro/azext_aro/custom.py

```diff
@@ -191,13 +211,70 @@ def aro_list_credentials(client, resource_group_name, resource_name):
     return client.list_credentials(resource_group_name, resource_name)
 
 
-def aro_update(client, resource_group_name, resource_name, no_wait=False):
-    oc = v2020_04_30.OpenShiftClusterUpdate()
+def aro_update(cmd, client, resource_group_name, resource_name, no_wait=False):
```

Comment:
a lot of the comments made on aro_delete will also apply to aro_update - please check them through.